.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Weekly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 00:00 on Monday
+    - cron: "0 0 * * 1"
+
+jobs:
+  bump-version-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
```

Comment:
What happens here if any other dev branch contains a more recent tag? Does it make sense to fetch history from the `main` branch only?

Reply:
Should be addressed by Federico's suggestion below.